StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/FormHeaderView.swift

```diff
@@ -16,15 +16,23 @@ final class FormHeaderView: UIView {
         let label = PaymentSheetUI.makeHeaderLabel(appearance: appearance)
         if paymentMethodType == .stripe(.card) {
             label.text = hasASavedCard ? String.Localized.add_card : String.Localized.add_new_card
+        } else if paymentMethodType == .stripe(.USBankAccount) {
+            label.text = String.Localized.add_us_bank_account
```

Comment:
Strange, this text seems to be bigger than `paymentMethodType.displayName`.  I think the icon height is constraining the text height.

Reply:
Seems to be tied to: 
```swift
    header.adjustsFontSizeToFitWidth = true
    header.adjustsFontForContentSizeCategory = true
```
when I comment these out the label is the correct size.